traefik/templates/_podtemplate.tpl

```diff
@@ -598,14 +598,25 @@
           {{- range $entrypoint, $config := $.Values.ports }}
           {{- if $config }}
             {{- if $config.redirectTo }}
-             {{- $toPort := index $.Values.ports $config.redirectTo.port }}
+              {{- fail "ERROR: redirectTo syntax has been removed in v34 of this Chart. See Release notes or EXAMPLES.md for new syntax." -}}
+            {{- end }}
+            {{- if $config.redirections }}
+             {{- with $config.redirections.entryPoint }}
+              {{- if and (eq .to "websecure") (ne .scheme "https") }}
```

Comment:
We can't assume users  didn't specify:

```yaml
ports:
  websecure:
    tls:
      enabled: false
```

Reply:
Could you just check if the port on which we redirect has `tls.enabled:  true` ?
We might also want to check if the `to` port exists.